test/e2e/tests/request-queuing/dapp1-send-dapp2-signTypedData.spec.js

```diff
@@ -44,17 +46,13 @@ describe('Request Queuing Dapp 1, Switch Tx -> Dapp 2 Send Tx', function () {
         title: this.test.fullTitle(),
       },
       async ({ driver }) => {
-        await unlockWallet(driver);
+        await loginWithBalanceValidation(driver);
 
         // Open and connect Dapp One
         await openDapp(driver, undefined, DAPP_URL);
 
-        await driver.findClickableElement({ text: 'Connect', tag: 'button' });
-        await driver.clickElement('#connectButton');
-
-        await driver.delay(regularDelayMs);
+        await driver.clickElement({ text: 'Connect', tag: 'button' });
 
-        await driver.waitUntilXWindowHandles(3);
```

Comment:
- we can omit all the find elements, before clicking those same selectors
- we can omit hardcoded delays
- we can omit the wait for x windows, before switching to a window by title